WcaOnRails/app/models/schedule_activity.rb

```diff
@@ -60,7 +60,7 @@ def localized_name(rounds_by_wcif_id = {})
     parts = ScheduleActivity.parse_activity_code(activity_code)
     if parts[:event_id] == "other"
       # TODO/NOTE: should we fix the name for event with predefined activity codes? (ie: those below but 'misc' and 'unofficial')
-      # VALID_OTHER_ACTIVITY_CODE = %w(registration breakfast lunch dinner awards unofficial misc).freeze
+      # VALID_OTHER_ACTIVITY_CODE = %w(registration checkin multi breakfast lunch dinner awards unofficial misc).freeze
```

Comment:
Anyone now why this is commented out and what this todo means in this context?

Reply:
I think the idea behind it was to provide a consistent translation for the most common "other" activities (eg: registrations, awards, lunch, etc); basically I didn't go through with it because I was unsure who we wanted this to appear and I think because it implied locking the "other" activity name input for some activities and it was not a priority in creating the schedule tool.

As for why this line is commented out I think it was just so that there was a reference to the current "other" activity code we know about ^^